Makefile

```diff
@@ -1,30 +1,13 @@
 include Makefile.build.mk
-CURRENT_USER := $(if $(GITHUB_USER),$(GITHUB_USER),$(shell id -u -n))
+CURRENT_USER := $(shell id -u -n)
 MINIO_TAG ?= 0.20240909.165928-r0
 RQLITE_TAG ?= 8.30.2-r0
 DEX_TAG ?= 2.41.1-r0
 LVP_TAG ?= v0.6.7
 PACT_PUBLISH_CONTRACT ?= false
 
-define sendMetrics
-@if [ -z "${PROJECT_NAME}" ]; then \
-    echo "PROJECT_NAME not defined"; \
-    exit 1; \
-fi
-@curl -X POST "https://api.datadoghq.com/api/v1/series" \
--H "Content-Type: text/json" \
--H "DD-API-KEY: ${DD_API_KEY}" \
--d "{\"series\": [{\"metric\": \"build.time\",\"points\": [[$$(date +%s), $$(expr $$(date +%s) - $$(cat start-time))]],\"tags\": [\"service:${PROJECT_NAME}\"]}]}"
-endef
-
-.PHONY: capture-start-time
-capture-start-time:
-	@echo $$(date +%s) > start-time
-
-.PHONY: report-metric
-report-metric:
-	@$(if ${DD_API_KEY}, $(call sendMetrics))
-	@rm start-time
+OS ?= linux
+ARCH ?= $(shell go env GOARCH)
```

Comment:
the go build commands do not respect these vars. is that intended?

Reply:
yes because the kots cli will typically run on your mac, not inside the container. the one inside the container gets built differently.